rfcs/select-component.md

```diff
@@ -150,7 +150,9 @@ export default () => {
   Lets you control what label is rendered for the specified option. The default behavior returns `option.label`.
 * `getSelectedOptionLabel?: (option: Object) => React$Node`:
   Lets you control what label is rendered for the selected option. If not specified, defaults to `getOptionLabel`.
-* `autoFocus: boolean`:
+* `disabled: boolean`:
+  Make the control disabled. Default is `false`
```

Comment:
It looks like we already list `disabled` here two lines below?

Reply:
Nice catch, removed.